tacc-wbomar/Core-Styles
# Version CLI

## Overview

Break CLI into two commands `version` (new) and `build` (previous) and polish everything.

## Related

### Required By

* https://github.com/TACC/Core-CMS/pull/448

## Changes

- Add `version` command to CLI.
- Make `build` be a command.
- Polish everything.

## Testing

1. Run these commands on Core-Styles:
    - `node cli.js`
    - `node cli.js build --help`
    - `node cli.js version --help`
2. Expect no errors (and no dynamic changes to repo files).
3. Run these commands on Core-CMS (after required changes\*):
    - `npm run build`
    - `npm run build --project=ecep-cms`
5. Expect no errors (and no dynamic changes to repo files).

<details><summary>* Required changes to Core-CMS…</summary>

- `core-styles …` → `core-styles build`
- `update-css-version.js` → `version-css-via-cli.js` (make it use `core-styles version…`)
- `--custom-config-files` → `--custom-configs`

</details>
